[gardener-local] [mac] Error while `make kind-ha-multi-zone-up` trying to add IPv6 address to lo0

**How to categorize this issue?**
<!--
Please select area, kind, and priority for this issue. This helps the community categorizing it.
Replace below TODOs or exchange the existing identifiers with those that fit best in your opinion.
If multiple identifiers make sense you can also state the commands multiple times, e.g.
  /area testing
  /kind auto-scaling
  ...

"/area" identifiers:     audit-logging|auto-scaling|backup|certification|control-plane-migration|control-plane|cost|delivery|dev-productivity|disaster-recovery|documentation|high-availability|logging|metering|monitoring|networking|open-source|ops-productivity|os|performance|quality|robustness|scalability|security|storage|testing|usability|user-management
"/kind" identifiers:     api-change|bug|cleanup|discussion|enhancement|epic|impediment|poc|post-mortem|question|regression|task|technical-debt|test
-->
  /area testing
/kind bug test

**What happened**:
Error occurred while `make kind-ha-multi-zone-up` using the script [./hack/kind-up.sh](https://github.com/gardener/gardener/blob/master/hack/kind-up.sh#L57-L64)  is trying to add an IPv6 address to the loopback device lo0 in mac OS.

``` bash
> make kind-ha-multi-zone-up

Checking loopback device lo0...
IP address 127.0.0.10 already assigned to lo0.
IP address 127.0.0.11 already assigned to lo0.
IP address 127.0.0.12 already assigned to lo0.
Adding IP address ::10 to lo0...
Executing: /usr/bin/sudo /sbin/ifconfig lo0 inet6 add ::10
ifconfig: ioctl (SIOCAIFADDR): Invalid argument
Usage: ip addr show [ dev STRING ]
       ip addr { add | del } PREFIX dev STRING
make: *** [kind-ha-multi-zone-up] Error 255
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:

- Gardener version:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Others:

--- a/hack/kind-up.sh
+++ b/hack/kind-up.sh
@@ -53,8 +53,12 @@ setup_loopback_device() {
     return
   fi
   LOOPBACK_DEVICE=$(ip address | grep LOOPBACK | sed "s/^[0-9]\+: //g" | awk '{print $1}' | sed "s/:$//g")
+  LOOPBACK_IP_ADDRESSES=(127.0.0.10 127.0.0.11 127.0.0.12)
+  if [[ "$IPFAMILY" == "ipv6" ]]; then
+    LOOPBACK_IP_ADDRESSES+=(::10 ::11 ::12)
+  fi
   echo "Checking loopback device ${LOOPBACK_DEVICE}..."
-  for address in 127.0.0.10 127.0.0.11 127.0.0.12 ::10 ::11 ::12; do
+  for address in ${LOOPBACK_IP_ADDRESSES[@]}; do
     if ip address show dev ${LOOPBACK_DEVICE} | grep -q $address; then
       echo "IP address $address already assigned to ${LOOPBACK_DEVICE}."
     else

